webapp/game/templates/test_game.html

```diff
@@ -3,11 +3,37 @@
 {% block content %}
   <h2>game</h2>
 
+  <form action="/suggest/" method="POST">
+    {% csrf_token %}
+
+    <label for="reset_quest_members">Who should go on this quest?</label>
+
+    <br><br>
+    {% for game_user in game_users %}
+      <input
+        type="checkbox"
+        id={{ game_user.pk }}
+        name="users"
+        value={{ game_user.pk }}>
+      <label for={{ game_user.pk }}>{{ game_user.user.username }}</label>
+      <br>
+    {% endfor %}
+    <br>
+
+    <input type="text" name="game_pk" value={{ pk }} hidden />
+    <input type="submit" value="Pick your crew" />
+  </form>
+
   <h3>current state</h3>
 
   current_quest: {{ current_quest.pk }}<br>
   quest_master: {{ quest_master.user.username }}<br>
-  quest_members: {{ current_quest.members }}<br>
+
+  quest_members:
+  {% for m in current_quest.members %}
+    <a>{{ m.member.user.username }}, </a>
```

Comment:
it may be worth to encapsulate username on AvalonQuestMember and AvalonGameUser models instead of manually daisy chaining